gravity-forms/gw-validate-that-a-value-exists.php

```diff
@@ -0,0 +1,364 @@
+<?php
+/**
+ * Gravity Wiz // Gravity Forms // Validate that a Value Exists
+ *
+ * Ensure that a value entered in Form A has been previously submitted on Form B. This is useful if you're generating a reference number of some sort
+ * on Form B and would like the user to enter it on Form A.
+ *
+ * @version   1.8
+ * @author    David Smith <david@gravitywiz.com>
+ * @license   GPL-2.0+
+ * @link      http://gravitywiz.com/...
```

Comment:
Let's update the @link to point to the article URL: https://gravitywiz.com/require-existing-value-submission-gravity-forms/